edsnlp/pipelines/qualifiers/negation/negation.py

```diff
@@ -252,7 +252,7 @@ def process(self, doc: Doc) -> Doc:
         # Removes duplicate matches and pseudo-expressions in one statement
         matches = filter_spans(matches, label_to_remove="pseudo")
 
-        entities = list(get_spans(doc, self.span_getter))
+        entities = list(get_spans(doc, self.span_getter or {"ents": True}))
```

Comment:
To mimick our old behaviour: when `on_ents_only` was set to True, we still qualified entities in `doc.ents`